scripts/shelley-testnet-dns.sh

```diff
@@ -24,9 +24,8 @@
 ALGO="--real-pbft"
 NOW=`date "+%Y-%m-%d 00:00:00"`
 NETARGS="--slot-duration 2 --topology configuration/simple-topology-dns.json ${ALGO}"
-#SCR="./scripts/start-node.sh"
 #CMD="stack exec --nix cardano-node --"
-CMD="cabal new-run exe:cardano-node --"
+CMD="cabal new-run exe:cardano-node -- mock-protocol"
```

Comment:
Is this for the testnet? Why is this a mock protocol?

Reply:
Same as: https://github.com/input-output-hk/cardano-node/pull/335#discussion_r351763574